presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -58,6 +58,7 @@
     QUERY_TEXT_TOO_LARGE(0x0000_0023, USER_ERROR),
     UNSUPPORTED_SUBQUERY(0x0000_0024, USER_ERROR),
     EXCEEDED_FUNCTION_MEMORY_LIMIT(0x0000_0025, USER_ERROR),
+    ADMINISTRATIVELY_KILLED(0x0000_0026, USER_ERROR),
```

Comment:
maybe `INSUFFICIENT_RESOURCES `

Reply:
I think ADMINISTRATIVELY_KILLED is better because it is just for the administrator to kill the query. It may not necessarily be INSUFFICIENT_RESOURCES.

Reply:
we already have a `GENERIC_INSUFFICIENT_RESOURCES` code, so `INSUFFICIENT_RESOURCES` will also be confusing.

Reply:
Maybe `KILLED_BY_ADMIN`? Administratively sounds weird too.

Reply:
We want this to be `USER_ERROR` and decided on the name `ADMINISTRATIVELY_KILLED`. Simply having `KILLED` is too vague. `KILLED_BY_ADMIN` sounds strange and is misleading, as it sounds like a human admin, when this could be used for any number of administrative reasons (automated systems which I wouldn't call an "admin").

This name is similar to the common software phrase *"administratively prohibited"*.